open_rarity/models/utils/attribute_utils.py

```diff
@@ -0,0 +1,17 @@
+def normalize_attribute_string(word: str) -> str:
```

Comment:
nit: word -> `attribute_string`, `literal or value` , `either works`.
nit: normalize_attribute_string -> normalize_string_attribute 
( this will help us with consistent naming for other types like dates or bitmaps )